packages/Calendar/src/index.js

```diff
@@ -1,3 +1,2 @@
 export { default as SingleDateCalendar } from "./SingleDateCalendar";
 export { default as DateRangeCalendar } from "./DateRangeCalendar";
-export * from "./tokens";
```

Comment:
Isn't this a breaking change?

Reply:
not really, there's no file called `tokens` so it never worked before